client/src/components/Form.js

```diff
@@ -0,0 +1,123 @@
+
+export default function Form({ submitFunction, open }) {
+
+    const userBrofile = {
+        bro_username: '',
+        bro_name: '',
+        bro_location: '',
+        image: ''
+    }
+    const [newBrofile, setNewBrofile] = useState(userBrofile);
+    const [valid, setValid] = useState(false)
+
+    const handleChange = event => {
+        const field = event.target;
+        const value = field.value;
+        setNewBrofile({
+            ...newBrofile,
+            [field.name]: value
+        })
+
+        function submitForm(event) {
+            event.preventDefault();
+            if (isValidBrofile(newBrofile)) {
+                setValid(true);
+                submitFunction(newBrofile);
+                setNewBrofile(userBrofile);
+            }
+        }
+        return (
+            <>
+                <FormWrapper open={open} valid={valid} onSubmit={submitForm}>
+                    <input
+                        type='text'
+                        name='bro_username'
+                        placeholder='Enter your desired Username'
+                        onChange={handleChange}
+                        value={newBrofile.bro_username} />
+
+                    <input
+                        type='text'
+                        name='bro_name'
+                        placeholder='Enter your full name'
+                        onChange={handleChange}
+                        value={newBrofile.bro_name} />
+                    <input
+                        type='text'
+                        name='bro_location'
+                        placeholder='Enter your city'
+                        onChange={handleChange}
+                        value={newBrofile.bro_location} />
+
+                    <label>Add image
+        </label>
+                    <input
+                        type='file'
+                        name='image'
+                        placeholder='Add image'
+                        onChange={handleChange}
+                        value={newBrofile.image}
+                    />
+                    <CreateBrofileButton valid={valid} type='submit'> Create Profile</CreateBrofileButton>
+                </FormWrapper>
+            </>
+        )
+    }
+}
+
+const FormWrapper = styled.form`
+display:flex;
+flex-direction: column;
+margin: 1.2rem 2.3rem;
+gap: 0.2rem;
+opacity: ${({ open, valid }) => open || valid ? '40%' : '100%'};
+position: absolute;
+
+input, select, textarea{
+    margin-bottom: 1rem;
+    border-radius: 1rem;
+    box-shadow: 0.1rem 0.2rem 0.2rem 0.1rem rgba(0,0,0, 35%);
+    border: none;
+    padding: 0.6rem;
+    background: white;
+    outline: none;
+    font-style: italic;
+    color: var(--mainwhite);
+    }
+
+    /*input:valid,
+select:valid,
+textarea:valid{
+  box-shadow: 0 0 5px 1px var(--mainwhite);
+}*/
+/*textarea{
+    height: 6rem;
+    padding: 0.8rem;
+}
+label{
+    margin-left: 0.5rem;
+    color: var(--mainwhite);
+}
+
```

Comment:
```suggestion
    /*input:valid,
select:valid,
textarea:valid{
  box-shadow: 0 0 5px 1px var(--mainwhite);
}*/
/*textarea{
    height: 6rem;
    padding: 0.8rem;
}
label{
    margin-left: 0.5rem;
    color: var(--mainwhite);
}
*/
```
Do not push commented out code into main. Don't be afraid to delete unused code, you will always find it in your git history! 
Also there was an issue here with the commented out code. You forgot to close the second comment. This caused for me that the whole form could not be rendered when I wanted to open the app

Reply:
This was also the issue why the Brofile.js was not loading for you ;)